algorithms/kernel/service_heap.h

```diff
@@ -35,6 +36,13 @@ namespace internal
 {
 using namespace services::internal;
 
+template <CpuType cpu, typename RandomAccessIterator>
+struct StandardComparator
```

Comment:
This kind of objects usually called DefaultSmth. In your case it will be DefaultComparator.